Modules/Package.swift

```diff
@@ -169,7 +170,12 @@ let package = Package(
         ),
         .testTarget(name: "JetpackStatsWidgetsCoreTests", dependencies: [.target(name: "JetpackStatsWidgetsCore")], swiftSettings: [.swiftLanguageMode(.v5)]),
         .testTarget(name: "DesignSystemTests", dependencies: [.target(name: "DesignSystem")], swiftSettings: [.swiftLanguageMode(.v5)]),
-        .testTarget(name: "WordPressFluxTests", dependencies: ["WordPressFlux"], swiftSettings: [.swiftLanguageMode(.v5)]),
+        .testTarget(
+            name: "WordPressFluxTests",
+            dependencies: ["WordPressFlux"],
+            exclude: ["WordPressFluxTests.xctestplan"],
+            swiftSettings: [.swiftLanguageMode(.v5)]
+        ),
```

Comment:
I added this to see how `WordPressFluxTests` would run tests with standalone scheme.

Interestingly, I tried using a copy-paste friendly `exclude: ["*.xctestplan"],` pattern, but it failed to find the file.

Reply:
My autogenerated scheme picks up the test plan:

<img width="956" alt="image" src="https://github.com/user-attachments/assets/f998752d-c899-42fc-81e0-c08c123fb8d1" />

However, I got an unlock prompt when I edited it (but no `xcsheme` change in the project, possibly because the scheme might be in `Modules/.swiftpm`) so I'd be curious to know if you see it too...